### PR TITLE
ForgotPasswordScreen props destructuring is a mess

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-4688 .

#### Changes proposed in this Pull Request:

- Redundant props removed in the ForgotPasswordScreen types.
- Updated the ForgotPasswordScreen component.

#### To Test:

- yarn start:example

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

File: login-workflow/src/screens/ForgotPasswordScreen/types.ts [hunk 1]
```diff
@@ -3,76 +3,39 @@ import { WorkflowCardProps } from '../../components/WorkflowCard/WorkflowCard.ty
...
-    /**
-     * Used to determine whether to show a success screen after the form is submitted
-     */
+    // used to display contact phone number
+    contactPhone?: string;
+
+    // used to display response time
+    responseTime?: string;
+
+    // TODO: These need cleaned up to use the WorkflowCardProps instead
```
Comment (ektaghag-eaton) [inline login-workflow/src/screens/ForgotPasswordScreen/types.ts:31]: Can we remove this?

File: login-workflow/src/screens/ForgotPasswordScreen/types.ts [hunk 2]
```diff
@@ -3,76 +3,37 @@ import { WorkflowCardProps } from '../../components/WorkflowCard/WorkflowCard.ty
...
+    // used for each slot in `ForgotPasswordScreenBase`
     slots?: {
         SuccessScreen?: (props: SuccessScreenProps) => JSX.Element;
     };
 
-    /**
-     * Applied to slot from SuccessScreen
-     */
+    // props applied to each slot
     slotProps?: {
```
Comment (JeffGreiner-eaton) [inline login-workflow/src/screens/ForgotPasswordScreen/types.ts:21]: update the documentation format to be similar to other type files

/**
     * A function used to test the input for valid formatting
     * @param {string} email - the provided email address
     * @returns boolean | string
     */
or 
/**
     * Applied to slot from SuccessScreen
     */

**Comment by JeffGreiner-eaton** [comment]
on my local the forgot password test is failing but has passed in CI?

image

**Comment by manojleaton** [comment]
> 

It's working fine in my local

**Comment by manojleaton** [comment]
> on my local the forgot password test is failing but has passed in CI?
> 
> image

It's working fine in my local

**Review by JeffGreiner-eaton — APPROVED** [review]
This looks good

**Pull request merged**